.github/workflows/release.yml

```diff
@@ -161,3 +161,22 @@ jobs:
           git add ftl.hcl
           git commit -m "Auto-versioned"
           git push origin main
+  vscode-release:
+    name: Release VSCode Extension
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Init Hermit
+        uses: cashapp/activate-hermit@v1
+      - name: Publish
+        run: |
+          version="$(git describe --tags --abbrev=0)"
+          echo "Publishing version $version"
+          jq ".version = \"$version\"" extensions/vscode/package.json > extensions/vscode/package.json.tmp
```

Comment:
Doesn't matter for this change, but it's preferable to use `jq --arg VERSION "$version" '.version = $VERSION'` to avoid escaping issues.